importScripts/sql/transformations/ObjectsExportView.sql

```diff
@@ -93,7 +93,9 @@ CREATE OR REPLACE VIEW ObjectsExport AS
         pc.Condition                                              AS PhysicalConditionCode,
         pc.ConditionDescription                                   AS PhysicalConditionDescription,
         # When SecondaryClass is blank then PrimaryClass's value is already in `Classification`
-        CASE WHEN
+        CASE WHEN o.ItemType = 'Photograph'
```

Comment:
The changes in this file are the actual change to the import, basically just returning `NULL` when the record is a Photograph.